### DISCUSSION TITLE
Attempting to install MISP - Error: fatal: not a git repository (or any of the parent directories): .git

### DISCUSSION THREAD

**Discussion by aumaxx** [MISP/MISP#9125] [Q&A]
I am attempting to install MISP on 22.04 and while installing I did get the error:

sudo: /var/www/MISP/venv/bin/pip: command not found;
apt is maybe locked, waiting 3 seconds.

This error was corrected with the following:

sudo virtualenv -p python3 /var/www/MISP/venv

However, now I am getting the following error.

Next step: Installing MISP Core
Next step: Trying to git pull existing install
fatal: not a git repository (or any of the parent directories): .git

debug:

fatal: not a git repository (or any of the parent directories): .git

[[ 128 -ne 0 ]]
sudo -H -u www-data git -C /var/www/MISP submodule update --progress --init --recursive

Thank you

**Comment by tfmarques23** [comment]
Hi,
Today i try install on ubuntu 22.04 and have the same problem...
